src/services/account.service.ts

```diff
@@ -0,0 +1,28 @@
+import { AccessToken, ApiResponse } from '../types';
+import { Account } from '../types/account';
+import { getAccessToken } from '../utils';
+import { APIService } from './api';
+
+export class AccountService extends APIService {
+  getAccountDetails = async (): Promise<ApiResponse<Account>> => {
+    const userAccessToken = getAccessToken() as AccessToken;
+
+    return this.get(`/accounts/${userAccessToken.accountId}`, {
```

Comment:
I think we need to create an instance of `ApiResponse`, as well as convert account json from service into JS object by instantiating `Account` class before returning a response.